src/5e-SRD-Features.json

```diff
@@ -71,36 +174,56 @@
       "Starting at 2nd level, you can throw aside all concern for defense to attack with fierce desperation. When you make your first attack on your turn, you can decide to attack recklessly. Doing so gives you advantage on melee weapon attack rolls using Strength during this turn, but attack rolls against you have advantage until your next turn."
     ],
     "feature_specific": {
-      "action": [
-        {
-          "type": "attack",
-          "attack_type": "melee",
-          "attack_modifier": "str",
-          "desc": "Advantage applies to melee attacks with STR modifier only"
-        }
-      ],
-      "prerequisites": [
-        {
-          "type": "equals",
-          "description": "attack is the first of the turn",
-          "value": {
-            "type": "integer",
-            "value": "turns.attacks_made",
-            "desc": "attacks made so far this turn"
-          },
-          "target": {
-            "type": "integer",
-            "value": 0
-          }
-        }
-      ],
-      "debuffs": [
+      "actions": [
         {
-          "type": "advantage",
-          "for": "enemy",
-          "on": "attack",
-          "duration": "2 turns",
-          "desc": "Attack rolls against you have advantage until the end of your next turn"
+          "name": "Reckless Attack",
+          "desc": "When you make your first attack on your turn...",
```

Comment:
This should be the full description of the action. From the SRD:

> When	you	make	your	first	attack	on	your	turn,	you	can	
decide	to	attack	recklessly.	Doing	so	gives	you	
advantage	on	melee	weapon	attack	rolls	using	
Strength	during	this	turn,	but	attack	rolls	against	
you	have	advantage	until	your	next	turn.